fitbook1/src/App copy.css

```diff
@@ -0,0 +1,9 @@
+.App {
+  display: flex;
+  justify-content: center;
+}
+.app-container{
+  display: grid;
+  grid-template-columns: 200px auto 300px;
```

Comment:
suggestion: can use rem instead of px

Reply:
I dont want it to change the width according to the root font size that is why i have kept it like this